docs/instruments/LORISLogicParser.md

```diff
@@ -0,0 +1,143 @@
+# LORIS Logic Parser
+
+- [JS Use](#js-use)
+- [PHP Use](#php-use)
+- [Development](#development)
+- [Syntax](#syntax)
+
+The parser provides a human readable syntax for front end equation building, real-time calculations, and back end validation. 
+
+The syntax can be used to determine whether instrument fields are shown, required, or hidden in survey mode.
+It can also be used to perform real-time scoring calculations while referencing other fields or subject meta-data.
+
+Fun features: infinite variable nesting, easily managed evaluation, human readable syntax (similar to Excel), AST (abstract syntax tree) evaluation, fraction factorials, variance and standard deviation, date calculations, boolean operations, type checking and much more.
```

Comment:
```
[..] human readable syntax (similar to Excel) [...]
```

which is it?

Reply:
are you suggesting that Excel is not human readable syntax? lol